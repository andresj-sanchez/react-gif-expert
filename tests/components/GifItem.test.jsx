import { render } from "@testing-library/react";
import { GifItem } from "../../src/components/GifItem";

describe('Pruebas en <GifItem />', () => {

	const title = 'One Punch Man GIF';
	const url = 'https://media1.giphy.com/media/VXJWhaO7afRe/giphy.gif?cid=ccfc8180dbuw2mst0bld3k6n1dqu9los0lbbu4nnh9b83hmn&rid=giphy.gif&ct=g';
	
	test('debe de hacer match con el snapshot', () => {
		const { container } = render( <GifItem title={ title } url={ url } /> );
		expect( container ).toMatchSnapshot();
	});
});