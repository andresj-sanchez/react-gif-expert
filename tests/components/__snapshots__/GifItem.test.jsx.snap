// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifItem /> debe de hacer match con el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="One Punch Man GIF"
      src="https://media1.giphy.com/media/VXJWhaO7afRe/giphy.gif?cid=ccfc8180dbuw2mst0bld3k6n1dqu9los0lbbu4nnh9b83hmn&rid=giphy.gif&ct=g"
    />
    <p>
      One Punch Man GIF
    </p>
  </div>
</div>
`;
